### PR TITLE
minor edit: Makefile

### PR DESCRIPTION
**Pull request by travispulley**
Suggested command for installing ninja build, depending on package manager

### CODE REVIEW

**Review by KlausEverWalkingDev — COMMENTED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -528,7 +528,7 @@ require:
 	@go version >/dev/null 2>&1 || (echo -e "ERROR: go is required."; exit 1)
 	@which aclocal > /dev/null || (echo -e  "ERROR: automake is required. Install with:\n\n    $(POSIX_PKG_MANAGER) install automake"; exit 1)
 	@which $(LIBTOOL) > /dev/null || (echo -e "ERROR: libtool is required. Install with:\n\n    $(POSIX_PKG_MANAGER) install libtool"; exit 1)
-	@which ninja > /dev/null || (echo -e "ERROR: Ninja is required. Install with:\n\n    $(POSIX_PKG_MANAGER) install ninja"; exit 1)
+	@which ninja > /dev/null || (echo -e "ERROR: Ninja is required. Install with:\n\n    $(POSIX_PKG_MANAGER) install ninja-build"; exit 1)
```
Comment (Jarred-Sumner) [inline Makefile]: This differs by platform - macOS it's `ninja` and on Ubuntu, I think it is `ninja-build`

Comment (travispulley) [reply Makefile]: This possibility crossed my mind, but I didn't go the distance to see if it was the case.
I ran into this while using Debian on WSL2.
I'm going to ponder a better solution for this and flag it for non-use for now

File: Makefile [hunk 2]
```diff
@@ -520,6 +520,12 @@ docker-push-base:
 	docker push ghcr.io/jarred-sumner/bun-base-with-args:latest
 
 require:
+	ifeq ($(POSIX_PKG_MANAGER), brew)
+		PKGNAME_NINJA = ninja
+	else
+		PKGNAME_NINJA = ninja-build
+	endif
+
```
Comment (travispulley) [inline Makefile]: seems verbose to do it this way, but I'm intending proper Makefile format

File: Makefile (hunk 1 above)
Comment (travispulley) [reply Makefile]: > This differs by platform - macOS it's `ninja` and on Ubuntu, I think it is `ninja-build`

I added a conditional that addresses both now

**Comment by travispulley** [comment]
This is now tested and working

**Comment by Jarred-Sumner** [comment]
Thanks

**Pull request merged**